e2e/e2etests/test_solana_deposit_and_call_revert_with_call_that_reverts.go

```diff
@@ -0,0 +1,81 @@
+package e2etests
+
+import (
+	"github.com/gagliardetto/solana-go"
+	"github.com/near/borsh-go"
+	"github.com/stretchr/testify/require"
+
+	testcontract "github.com/zeta-chain/node/e2e/contracts/reverter"
+	"github.com/zeta-chain/node/e2e/runner"
+	"github.com/zeta-chain/node/e2e/utils"
+	solanacontracts "github.com/zeta-chain/node/pkg/contracts/solana"
+	crosschaintypes "github.com/zeta-chain/node/x/crosschain/types"
+)
+
+// TestSolanaDepositAndCallRevertWithCallThatReverts tests deposit of lamports
+// with revert options when call on revert program revets
```

Comment:
```suggestion
// with revert options when call on revert program reverts
```
Typo, can we also precise that it is supposed to abort